src/Todo/feature/setPriority.js

```diff
@@ -0,0 +1,62 @@
+import { api } from '../../etc/api.js';
+import { template } from '../../etc/template.js';
+import { getItemId, getMemberId, teamId } from '../todo.js';
+
+export const setPriority = async (target) => {
+  const memberId = getMemberId(target);
+  const itemId = getItemId(target);
+  const contents = {
+    priority: 'NONE',
+  };
+  const value = target.value;
+
+  if (value === '0') {
+    contents.priority = 'NONE';
+  } else if (value === '1') {
+    contents.priority = 'FIRST';
+  } else if (value === '2') {
+    contents.priority = 'SECOND';
+  } else {
+    resetSelector(target);
+    return;
+  }
```

Comment:
```suggestion
const priorities = {
  0 : 'NONE',
  1 : 'FIRST',
  2 : 'SECOND', 
}

contents.priority = priorities[value]
```

1주차 미션 피드백에 있는 Object Literal 을 활용하면 훨씬 깔끔하게 코드 작성이 가능할거 같아요!